src/test/java/org/reactome/release/orthopairs/OrthopairFileGeneratorTest.java

```diff
@@ -86,7 +82,7 @@ public void createSpeciesGeneProteinFileTest() throws IOException {
     }
 
     @Test
-    public void createSpeciesGeneProteinFileNullEnsemblIdsTest() throws IOException {
+    public void createSpeciesGeneProteinFileNullEnsemblIdsTest() throws NoSuchFileException, IOException {
```

Comment:
Why did this need to be added?

Reply:
It didn't - I was hoping to remove it and push the commit before you saw it :)